includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -651,10 +732,16 @@ protected function upload_file( $field_key, $field ) {
 			}
 
 			if ( ! empty( $field['multiple'] ) ) {
-				return $file_urls;
+				$value = $file_urls;
 			} else {
-				return current( $file_urls );
+				$value = current( $file_urls );
 			}
 		}
+
+		if ( isset( $field['before_sanitize'] ) && is_callable( $field['before_sanitize'] ) ) {
```

Comment:
TBH this doesn't do much and is called in a weird spot, but just thought I'd add it for consistency.